strcalc.R

```diff
@@ -1,10 +1,10 @@
-compute = function(input_string) {
+compute <- function(input_string) {
   # Do simple arithmetic from strings
   # Example: compute('5 + 2') -> 7
   values = unlist(strsplit(input_string, ' '))
```

Comment:
Nice! Thanks for the help, I'm new at this! But you seem to have missed one, did you do that on purpose?